easybuild/easyconfigs/s/snpEff/snpEff-3.6-Java-1.7.0_80.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PackedBinary'
+
+name = 'snpEff'
+version = '3.6'
+
+homepage = 'http://sourceforge.net/projects/snpeff/'
+description = """SnpEff is a variant annotation and effect prediction tool. 
+It annotates and predicts the effects of genetic variants (such as amino acid changes)."""
```

Comment:
1 leading space helps readability (we should be doing it across all easyconfigs, yeah)